rebar.config

```diff
@@ -166,6 +163,7 @@
                                   {d, deprecated_now},
                                   {d, deprecated_19}]}]},
    {override, hamcrest, [{plugins, []}]},
+   {override, eper, [{erl_opts, [debug_info]}]},
```

Comment:
Why do we need this?

Reply:
Because `eper` has `warning_as_errors` in its `rebar.config`. Also eper uses the now deprecated `erlang:get_stacktrace`. Without this override, `eper` simply fails to compile.

Reply:
Just as food for thought: since rebar3 3.6.0 (current MIM version is 3.5.0) there is a `del` option which allows the removal of flags in a more explicit way. For MIM you might even use
```
{overrides, [
    {del, [{erl_opts, [warning_as_errors]}]}
]}.
```
to get rid of this issue for all apps, especially since erlang is currently depricating functions with every release.

Reply:
Thanks for the tip @GalaxyGorilla I've updated rebar to 3.6.0 in this PR so I'm going to use the `del` option you mentioned.